TypeError: realpath is not a function

Using node v12.18.2 or v13.13.0, I'm getting the follow error:

```shell
(node:47407) UnhandledPromiseRejectionWarning: TypeError: realpath is not a function
    at visitDirectory (/Users/aneil/code/test-ts-auto-guard/node_modules/typescript/lib/typescript.js:18327:45)
    at Object.matchFiles (/Users/aneil/code/test-ts-auto-guard/node_modules/typescript/lib/typescript.js:18323:13)
...
    at Object.parseJsonConfigFileContent (/Users/aneil/code/test-ts-auto-guard/node_modules/typescript/lib/typescript.js:28544:16)
(node:47407) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

Replicate with a folder containing the following files:

* `tsconfig.json`
```json
{
  "compilerOptions": {
    "baseUrl": "./src",
    "outDir": "./lib",

  },
  "include": ["index.ts"],
  "exclude": ["node_modules", "**/__tests__/*"]
}
```

* `index.ts`
```typescript
/** @see {isMyType} ts-auto-guard:type-guard */
type MyType = string | number;
```

* `package.json`
```json
{
  "name": "test-ts-auto-guard",
  "version": "1.0.0",
  "main": "lib/index.js",
  "types": "lib/index.d.ts",
  "scripts": {
    "build": "tsc"
  },
  "dependencies": {
  },
  "devDependencies": {
    "ts-auto-guard": "^1.0.0-alpha.1"
  }
}
```

I get the error with:
```shell
npm install
node_modules/.bin/ts-auto-guard
```

Thanks - I changed my devDependencies in `package.json` to:

```json
"devDependencies": {
    "ts-auto-guard": "rhys-vdw/ts-auto-guard#master"
  }
```

And wiped out node_modules and ran `npm i`, but I'm getting this error:

```shell
npm WARN test-ts-auto-guard@1.0.0 No description
npm WARN test-ts-auto-guard@1.0.0 No repository field.
npm WARN test-ts-auto-guard@1.0.0 No license field.

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /Users/aneil/code/test-ts-auto-guard/node_modules/ts-auto-guard/lib/cli.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/aneil/code/test-ts-auto-guard/node_modules/ts-auto-guard/lib/cli.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "NODE_ENV=test && yarn run format:check && tape -r ts-node/register tests/**/*.ts | tap-diff",
     "build": "tsc && chmod 775 lib/cli.js",
-    "prepublishOnly": "npm run build",
+    "prepare": "npm run build",
     "format": "tslint --fix --project ./tsconfig.json && prettier --write src/**/*.ts",
     "format:check": "prettier --list-different src/**/*.ts"
   },

